src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -705,6 +705,13 @@ paths:
       operationId: exportSnapshot
       parameters:
         - $ref: '#/components/parameters/Id'
+        - in: query
```

Comment:
I know this was already in from a previous PR but I don't love that this is a `GET`.  IMO there shouldn't ever be a side effect to a get request.  It feels odd to have a get request trigger a job

Reply:
There are frequently side-effects to a `GET` request. The `enumerate` endpoints allow you to sort, filter, and provide dataset IDs, all with `GET` parameters. In our `retrieve` endpoints, `GET` parameters allow you to decide what information you want returned in the response like `ACCESS_INFORMATION`. If the contents of the parameter were being stored, I would agree that this should be a `POST`, but in this case, I think `GET` makes more sense.

Reply:
Those aren't side effects.  They're not modifying underlying state whereas this is.  It's creating files

Reply:
But the parameter is modifying the behavior of how the files are created, not being included in the files themselves. If there was no parameter for this, would you still be arguing for a `POST` request with an empty body?

Reply:
Yep exactly. I think empty body post are legit (though I'd also argue that the options should go in that body too)

Reply:
I feel like an empty post body introduces extra complexity for no benefit. What is that empty post body? An empty string? And empty JSON object? Null? Empty XML tag? A `GET` removes these ambiguities. Yeah, a user could read the documentation, but why should they have to if the empty body is providing no benefit?

Reply:
I just think that an API that creates something (e.g. an export) should not be GET request.  I actually find that more confusing (and again, I would have put the gs path option in there)

Reply:
I will concede, for the public record, that a `POST` request is probably a better choice, but for the sake of getting this merged, not change it now.